src/administration/Administration.Service/Controllers/ConnectorsController.cs

```diff
@@ -242,11 +242,11 @@ public async Task<IActionResult> DeleteConnectorAsync([FromRoute] Guid connector
     }
 
     /// <summary>
-    /// Retrieve company connector end point with bpn.
+    /// post list of bpn and empty body to Retrieve available company connector.
```

Comment:
the description is a bit confusing since you have to post a body, but the body could be an empty array.

@jjeroch should posting an empty body be possible?

```suggestion
    /// post list of bpns or an empty array to retrieve available company connector.
```